libs/framework/gtest/src/DependencyManagerTestSuite.cc

```diff
@@ -638,4 +639,12 @@ TEST_F(DependencyManagerTestSuite, ExceptionsInLifecycle) {
         //required service -> should stop, but fails at stop and should become inactive (component will disable itself)
         dm.clear(); //dm clear will deinit component and this should fail, but not deadlock
     }
+}
+
+TEST_F(DependencyManagerTestSuite, ComponentContext) {
```

Comment:
Please rename `ComponentContext` to `ComponentContextShouldNotLeak`.

Reply:
same